samples/sample_plugins/vpp_plugin/include/mfx_multi_vpp.h

```diff
@@ -38,8 +38,8 @@ class MFXVideoMultiVPP : public MFXVideoVPP
     virtual mfxStatus Init(mfxVideoParam *par, mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
     { par1; par2; return MFXVideoVPP_Init(m_session, par); }
 
-    virtual mfxStatus Reset(mfxVideoParam *par, mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
-    { par1; par2; return MFXVideoVPP_Reset(m_session, par); }
+    virtual mfxStatus Reset(mfxVideoParam *par)
```

Comment:
to avoid such errors in future, you can add "override" specifier

Reply:
Why? par1 and par2 do not used anywhere.

Reply:
Anyway, there is something ugly in overloading functions with a different number of parameters and using default values to get around problems.